docs/csharp/whats-new/csharp-7.md

```diff
@@ -361,17 +361,15 @@ modified. The local variable has been declared with the `ref` modifier,
 and it will take a `ref` return. You must initialize a `ref` variable when
 it is declared, you cannot split the declaration and the initialization.
 
-The C# language has two other rules that protect you from misusing
+The C# language has three other rules that protect you from misusing
 the `ref` locals and returns:
 
 * You cannot assign a value to a `ref` variable.
```

Comment:
Sorry to belatedly add this, @GuardRex. I think that this sentence makes sense only if you know what it means. It might be better to rephrase it as: "You cannot assign a standard method return value to a ref local variable." 
Otherwise, this looks good, unless @svick would like to see some further changes.